rest/jetty/src/test/scala/io/udash/rest/jetty/JettyRestCallTest.scala

```diff
@@ -1,11 +1,15 @@
 package io.udash
 package rest.jetty
 
-import io.udash.rest.raw.RawRest.HandleRequest
-import io.udash.rest.{RestApiTestScenarios, ServletBasedRestApiTest}
+import io.udash.rest.raw.RawRest
+import io.udash.rest.{RestApiTestScenarios, ServletBasedRestApiTest, StreamingRestApiTestScenarios}
 import org.eclipse.jetty.client.HttpClient
 
-final class JettyRestCallTest extends ServletBasedRestApiTest with RestApiTestScenarios {
+final class JettyRestCallTest
```

Comment:
Consider adding a testcase for calling streaming method with non-streaming client implementation